app/src/docs/_examples/dropdownMenu/Basic.example.tsx

```diff
@@ -107,7 +108,15 @@ export default function BasicDropdownMenuExample() {
                 closeOnMouseLeave="boundary"
                 closeDelay={ 400 }
                 renderBody={ (props) => renderDropdownBody(props) }
-                renderTarget={ (props) => <Avatar img="https://avatars.dicebear.com/api/human/avatar12.svg?background=%23EBEDF5&radius=50" size="36" { ...props } /> }
+                renderTarget={ (props) => (
+                    <LinkButton
+                        icon={ PersonIcon }
+                        caption="Alex Smith"
+                        size="36"
+                        { ...props }
+                        rawProps={ { 'aria-haspopup': props.isDropdown, 'aria-expanded': props.isOpen } }
```

Comment:
I guess we can add this attributes inside to the Button component, that they will be set by default if component act as dropdown.

Reply:
The current implementation of `ButtonBase` inheritance makes it impossible to achieve this (we will need to rework it in the future). As a solution, I suggest adding a `getProps` method to the `ButtonBase` that allows you to pass accessibility attributes from the `Button` component.